src/protocols/secure_channel/tests/TestCheckinMsg.cpp

```diff
@@ -137,27 +157,38 @@ void TestCheckin_Parse(nlTestSuite * inSuite, void * inContext)
     userData                         = chip::ByteSpan(data);
     const ccm_128_test_vector & test = *ccm_128_test_vectors[0];
 
-    Symmetric128BitsKeyByteArray keyMaterial;
-    memcpy(keyMaterial, test.key, test.key_len);
+    // Two disctint key material buffers to force the PSA unit tests to create two different Key IDs
```

Comment:
See comment I made about this comment, above.

Reply:
done